mount.go

```diff
@@ -98,15 +99,23 @@ func mount(source, destination, fsType string, flags int, options string) error
 	if fsType != type9pFs {
 		var err error
 
-		absSource, err = filepath.EvalSymlinks(source)
-		if err != nil {
-			return grpcStatus.Errorf(codes.Internal, "Could not resolve symlink for source %v", source)
+		if fsType == typeTmpFs {
```

Comment:
Since we have more than a couple cases now for the fsType now, I wonder if it'd warrant throwing this into a case now?  WDYT?

Reply:
Yes, `switch case` makes more sense here.